### PR TITLE
Feature/download link browser detection

### PR DESCRIPTION
**Pull request by mradkov**
- detect browser
    - different download url based on browser
    - TODO: update chrome url when published
- update AEX2 name to `Superhero`

fixes: https://github.com/aeternity/aepp-raendom/issues/44

### CODE REVIEW

File: src/components/layout/RightSection.vue [hunk 1]
```diff
@@ -55,10 +56,26 @@
...
     data() {
-      return {}
+      return { 
+        browser: detect()
+      }
     },
     computed: {
       ...mapGetters(['topics', 'isLoggedIn']),
+      downloadUrl () {
+        switch(this.browser && this.browser.name) {
```
Comment (thepiwo) [inline src/components/layout/RightSection.vue]: this shouldn't work, switch on a boolean?

Comment (mradkov) [reply src/components/layout/RightSection.vue]: It works actually, but if you are not happy with this - I can change the implementation.

Comment (kenodressel) [reply src/components/layout/RightSection.vue]: just wrap it in an if(this.browser) :)

File: src/components/layout/RightSection.vue [hunk 2]
```diff
@@ -7,7 +7,7 @@
           Wallet
         </div>
         <div class="section__body clearfix">
-          <div class="side__button"><a href="//github.com/superherowallet/wallet/releases/latest">Install Wallet</a></div>
+          <div class="side__button"><a :href="downloadUrl">Install Wallet</a></div>
```
Comment (thepiwo) [inline src/components/layout/RightSection.vue]: it should probably open in a new tab

**Pull request merged**